packages/core-connector-interface/src/types/acknowledgedTicket.d.ts

```diff
@@ -11,7 +11,7 @@ declare interface AcknowledgedTicketStatic {
   ): AcknowledgedTicket
 }
 
-declare interface AcknowledgedTicket {
+declare interface AcknowledgedTicket extends Uint8Array {
```

Comment:
Why is this included?

Reply:
Node 14 comes with slightly different types